.stylelintrc.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
         'shorthand-property-no-redundant-values': true,
         'string-no-newline': true,
         'time-min-milliseconds': 100,
-        'unit-allowed-list': ['fr', 'px', 'em', 'rem', '%', 'svh', 'vw', 'vh', 'deg', 'ms', 's', 'dpcm'],
```

Comment:
@farhan-nurzi-deriv I've changed all svh values in the deriv-app, so suppose we will not face the issue

<img width="441" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/ca676570-0c72-4129-8ebe-631d12dddf66">

Reply:
![image](https://github.com/binary-com/deriv-app/assets/125247833/100bdd67-1b35-42a5-8e04-d6cc20fcfb78)

Reply:
@farhan-nurzi-deriv by strange way I skipped this, good catch. I can get back svh to `unit-allowed-list` but in this case it doesn't have any sense, because goal of this PR is removing all svh from the app. This PR works just for wallet package so I need to move the logic to the core package I suppose and change the code in cashier, but in this case it will need QA attention :) need time to think about this

Reply:
@farhan-nurzi-deriv got back svh to `unit-allowed-list` but in the future I guess we'll need to refactor cashier package too to avoid using svh but this is out of the scope of this task